src/components/ArrowIndicator/ArrowIndicator.module.scss

```diff
@@ -0,0 +1,7 @@
+.arrowSvg {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  z-index: 10;
+  pointer-events: none;
+}
```

Comment:
Is this file necessary?

```suggestion
```

Reply:
I think so.. or at least the stylings it contains...